helm/cluster-openstack/templates/_kubeadm_control_plane_template.tpl

```diff
@@ -43,10 +43,4 @@ spec:
           {{- include "sshPostKubeadmCommands" . | nindent 10 }}
         users:
           {{- include "sshUsers" . | nindent 10 }}
-      machineTemplate:
-        infrastructureRef:
-          apiVersion: infrastructure.cluster.x-k8s.io/v1alpha4
-          kind: OpenStackMachineTemplate
-          name: {{ include "resource.default.name" . }}-{{ .Values.controlPlane.class }}
-      version: {{ .Values.kubernetesVersion }}
```

Comment:
Is the version set anywhere now?

Reply:
I think it's derived from the `Cluster` CR.

Reply:
Yes. It is declared here https://github.com/giantswarm/cluster-openstack/blob/main/helm/cluster-openstack/templates/_cluster.tpl#L15